docs/supervisor-tasks/README.md

```diff
@@ -1,97 +1,62 @@
-It's direct access to your **to-do list**. Per default, it's sorted by priority, so you don't have to think twice about what to do first; it's the one on top of the list.
-
-On the My Tasks page, you can filter the tasks list by Production, Task Type, Status, etc. 
+On the My Tasks page, you can filter the task list by Production, Task Type, Status, and other criteria.
 
 ![Supervisor Todo Page](../img/getting-started/supervisor_todo_page.png)
 
-
-Then, you must click on the **Status** to open the comment panel. From here, you can comment and publish a new revision.
+To view more details about a task, click on its **Status** to open the comment panel. Here, you can comment on the task and publish a new revision if needed.
 
 ![Supervisor Todo Detailed Page](../img/getting-started/supervisor_todo_page_detail.png)
 
-You can also use the Board view to drag and drop your task onto the new status column.
-
+You can also use the Board to quickly update the status of multiple tasks by dragging and dropping into each status column.
 
 ![Supervisor Todo Detailed Page board](../img/getting-started/supervisor_todo_page_detail_board.png)
 
+## Check the Previews of Your Department
 
+Now that everyone in your team knows what to do, you need to review their work. Typically, when an artist finishes a task, they will call you for a review. Frequent interruptions can be unproductive and make it difficult to maintain focus. To avoid this, set a specific timeframe for reviews. This approach benefits both you and your team by providing a structured schedule. Artists will know when they can expect feedback and can plan their work accordingly, while you can dedicate uninterrupted time to your own tasks.
 
-## Check the Previews of your Department
-
-Now that everyone in your team knows what to do, you must check their work.
-
-Usually, when an artist has finished a task, he will call you and ask for your review.
-So it means you must stop what you're doing, lose your focus, open their preview, focus on it, comment, and try to get back where you're stopped.
-
-It's a lot of wasted energy here, and you could be more productive, no thanks to all the intrusion.
-
-And the more you are interrupted, the longer it takes to regain focus.
-
-That's why we suggest you set a specific timeframe for your checking. The first advantage is that it will help your team be better organized. The artists will know when it's a good time to speak with you, and they can post their work and start a new task, knowing you will check it.
-The second advantage is that you can focus on your work! You know when the checking is and when to do your tasks. 
-
-If you click the **My Checks** button at the top of the screen, you will see the list of all your department's **WFA** tasks.
-
-![Supervisor My Chceck Page](../img/getting-started/supervisor_mycheck.png)
+<!-- Is this neccessary here, since it doesn't relate to Kitsu? -->
```

Comment:
It's interesting to give context.

Reply:
Agreed! Just wanted to check. I've removed the comment